app/views/shinycms/tags/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<% @content_types.each do |type| %>
+  <h2>
+    <%= type.underscore.pluralize.humanize %> tagged with '<%= @tag %>'
+  </h2>
+  <ul>
+    <% @tagged_items[ type ].each do |item| %>
+    <li>
+      <% if type == 'BlogPost' %>
```

Comment:
(Eventually) This might be a job for `polymorphic_path` to save the type checking code growing here when generating the URL: https://api.rubyonrails.org/classes/ActionDispatch/Routing/PolymorphicRoutes.html

```diff
-<%= link_to item.title, view_blog_post_path( item ) %>
+<%= link_to item.title, polymorphic_path( item ) %>
```